ci-operator/step-registry/lp-interop-tooling/archive-results/README.md

```diff
@@ -0,0 +1,35 @@
+# lp-interop-tooling-archive-results-ref<!-- omit from toc -->
```

Comment:
I have mixed feelings about creating this ref.

As of right now its not really needed since we are not reporting with report portal. My first thought is can we leave this out and create it once/if we need to.

Another thought I'm having is that I'm encouraging PQE to make changes to these files when they need to, archiving something new for their debugging purposes I think is something that we'll see pretty often. So I'm thinking that taking the simplest approach to that might be best. To me that is to just copy any results generated from tests into the $ARTIFACT_DIR as the last line of the commands.sh files.

I'm curious to here your thoughts.

Reply:
I think I agree with you here. It's an easy enough ref to recreate if we need to. I think I'll remove this and just archive the results in the execution ref!

Reply:
Resolved in commit [4602739](https://github.com/openshift/release/pull/36181/commits/4602739b55370b75da05c3fc8352787545510f44)